yellow-paper/docs/contract-deployment/classes.md

```diff
@@ -12,7 +12,7 @@ Read the following discussions for additional context:
 - [Abstracting contract deployment](https://forum.aztec.network/t/proposal-abstracting-contract-deployment/2576)
 - [Implementing contract upgrades](https://forum.aztec.network/t/implementing-contract-upgrades/2570)
 - [Contract classes, upgrades, and default accounts](https://forum.aztec.network/t/contract-classes-upgrades-and-default-accounts/433)
-  :::
```

Comment:
Is there an auto-formatter I should be using? Because my vscode chose to auto-format these spaces into existence in a previous PR...